action.yml

```diff
@@ -40,6 +40,11 @@ inputs:
       If false, the action will fail if an artifact for the given name already exists.
       Does not fail if the artifact does not exist.
     default: 'false'
+  include-hidden-files:
+    description: >
+      If true, hidden files will be included in the merged artifact.
+      If false, hidden files will be excluded from the merged artifact.
```

Comment:
What about just `uploaded` vs `merged`?

The main action is different from the other merge action

Reply:
Kept it simple and removed `merged` - https://github.com/actions/upload-artifact/pull/598/commits/710f36207581d49e465ccbed3e007c317290fe11